src/main/java/org/alliancegenome/curation_api/jobs/executors/ParalogyExecutor.java

```diff
@@ -94,5 +93,15 @@ private void runLoad(BulkLoadFileHistory history, String dataProvider, ParalogyI
 		ph.finishProcess();
 
 	}
+	
+	public APIResponse runLoad(String dataProvider, ParalogyIngestFmsDTO paralogyData) {
```

Comment:
@VarunReddy1111 you shouldn't need this as the super class should be able to take care of it.